### PR TITLE
Issue #535: Add admin endpoint for uptime

### PR DESCRIPTION
**Pull request by mikkokar**
Adds an `/admin/uptime` endpoint to show Styx uptime.

### CODE REVIEW

File: components/proxy/src/main/java/com/hotels/styx/admin/handlers/UptimeHandler.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+public class UptimeHandler implements WebServiceHandler {
+    private final MetricRegistry metricRegistry;
+
+    public UptimeHandler(MetricRegistry metricRegistry) {
+        this.metricRegistry = metricRegistry;
+    }
+
+    @Override
+    public Eventual<HttpResponse> handle(HttpRequest request, HttpInterceptor.Context context) {
+        Object uptime = metricRegistry.getGauges().get("jvm.uptime.formatted").getValue();
```
Comment (dvlato) [inline components/proxy/src/main/java/com/hotels/styx/admin/handlers/UptimeHandler.java]: Don't we need to check for null values here? I don't know if there are possible race conditions.

File: components/proxy/src/main/java/com/hotels/styx/admin/handlers/UptimeHandler.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+        return Eventual.of(HttpResponse.response(OK)
+                .disableCaching()
+                .addHeader(CONTENT_TYPE, "application/json")
+                .body(format("\"%s\"", uptime), UTF_8)
```
Comment (dvlato) [inline components/proxy/src/main/java/com/hotels/styx/admin/handlers/UptimeHandler.java]: Is this json content?

Comment (mikkokar) [reply components/proxy/src/main/java/com/hotels/styx/admin/handlers/UptimeHandler.java]: Hi. @dvlato, @chrisgresty said it is ;-)

Comment (chrisgresty) [reply components/proxy/src/main/java/com/hotels/styx/admin/handlers/UptimeHandler.java]: :)
JSON used to insist that only a map or an array was a valid document, but a few years ago this was changed to allow other value types (string, integer, etc). You could consider using `objectMapper.writeValueAsString(uptime)` instead.

Comment (dvlato) [reply components/proxy/src/main/java/com/hotels/styx/admin/handlers/UptimeHandler.java]: I guess technically we can say returning a quoted string is returning valid JSON... :)

**Pull request merged**